### PR TITLE
Added color support.

### PR DESCRIPTION
**Pull request by Dartellum**

### CODE REVIEW

**Comment by Sarvatt** [comment]
This change doesn't work with uncolored sanowrets, *.* instead of .*. does

http://rubular.com/r/udQCckkAtC

**Comment by Sarvatt** [comment]
Working fine here

File: sanowret-crystal.lic [hunk 1]
```diff
@@ -41,15 +41,15 @@ class SanowretCrystal
     if @worn_crystal
       use_crystal
     else
-      case DRC.bput('tap my sanowret crystal', /^You tap a .*. sanowret crystal inside your .*.$/, /^You tap a .*. sanowret crystal that you are wearing.$/, /^You tap a .*. sanowret crystal that you are holding.$/, 'I could not find what you were referring to.')
-      when /You tap a .*. sanowret crystal inside your/
+      case DRC.bput('tap my sanowret crystal', /^You tap a .*sanowret crystal inside your .*.$/, /^You tap a .*sanowret crystal that you are wearing.$/, /^You tap a .*sanowret crystal that you are holding.$/, 'I could not find what you were referring to.')
+      when /You tap a *.* sanowret crystal inside your/
```
Comment (Sarvatt) [inline sanowret-crystal.lic]: when /You tap a .*sanowret crystal inside your/

**Comment by Dartellum** [comment]
Fixed it.

**Comment by Sarvatt** [comment]
Working good now, thanks!

**Pull request merged**